File: 8kyu/(8 kyu) Well of Ideas - Easy Version/(8 kyu) Well of Ideas - Easy Version.go
package kata

func Well(x []string) string {
	goodIdeas := 0

	for _, w := range x {
		if w == "good" {
			goodIdeas += 1
		}
	}

	if goodIdeas == 0 {
		return "Fail!"
	} else if goodIdeas > 0 && goodIdeas < 3 {
		return "Publish!"
	} else {
		return "I smell a series!"
	}
}

File: 8kyu/(8 kyu) Well of Ideas - Easy Version/(8 kyu) Well of Ideas - Easy Version.cs
public class Kata
{
    public static string Well(string[] x)
    {
        int goodIdeas = 0;

        foreach (var idea in x)
        {
        if (idea == "good") {
            goodIdeas++;
        }
        if (goodIdeas > 2)
        {
            return "I smell a series!";
        }
        }

        if (goodIdeas == 0)
        {
        return "Fail!";
        }

        return "Publish!";
    }
}